bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+As a result of this assignment, the block producers will no longer provide feeds for BitAssets, and the new set of feed producers will no longer be affected by direct stakeholder voting.
+
+Because BitAsset management will no longer be affected by stakeholder voting, the vote-weighting calculations can be amended without detriment so that accounts holding BTS collateral positions will count only collateral above the value of their debt at the feed price. Collateral up to a 1.0 collateral-to-debt ratio is under lien to the BitAsset holder, and therefore does not represent a stake in the blockchain, and will no longer be counted as voting stake. Collateral in excess of a 1.0 collateral-to-debt ratio will continue to be counted in the debt holder’s voting stake.
+
+# Specification
+The blockchain software is designed to support assets with multisignature issuer authorities; therefore, the technical changes required for this proposal are straightforward. The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, the committee-controlled BitAsset objects' issuer fields will be updated to specify the cn-vote account (ID 1.2.1148184). The existing price feeds will be cleared, and the cn-vote account will be specified as the feed producer. After the hardfork, the cn-vote account will be able to assign other feed producers at its discretion.
+
+**Disable the witness_fed_asset and committee_fed_asset Flags**
+
+At the hardfork date, the `witness_fed_asset` and `committee_fed_asset` flags will be unset from the BitAssets, and will be disabled from use in future assets.
+
+**Amend total_core_in_orders to Track Limit and Collateral Orders Separately**
+
+The `total_core_in_orders` field of the `account_statistics_object` will be deprecated, and assets in limit orders will be tallied in a new `total_core_in_limit_orders` field while collateral will be tracked in a new `total_core_in_collateral_orders` map, keyed on the collateralized asset ID. During vote weight calculation, the `total_core_in_limit_orders` will add directly to the account’s voting weight; additionally, the `total_core_in_collateral_orders` which exceeds the feed price value of the collateralized debt will be added to the account’s voting weight as well.
+
+**Enable Remittance of Worker Proposals in Core Asset**
+
+Escrow agents administering existing worker proposals presently denominated in BitAssets shall be authorized to remit worker payments in BitShares core asset, or in a BitAsset selected at the discretion of the workers and escrow agents.
```

Comment:
What does this have to do with your proposal? They were always free to do this, no?

Reply:
Ultimately each escrow agent has final authority over remittances Andy hey typically desire to remain outside of governance issues. The intent here is to allow the Community to signal to escrow agents that changes to BItAssets mechanics after the initial worker proposal agreements were formed should be considered for future remittances.

Reply:
> They were always free to do this, no?

All (to my knowledge) *existing* worker proposals specify payout in BitAssets. Some escrows have refused payout in BTS because it contradicts the wording of the proposals.